wp-tevko-responsive-images.php

```diff
@@ -297,31 +297,130 @@ function tevkori_get_src_sizes( $id, $size = 'thumbnail' ) {
 }
 
 /**
- * Filter for extending image tag to include srcset attribute.
+ * Filters images in post content to add `srcset` and `sizes`.
  *
- * @see images_send_to_editor
- * @return string HTML for image.
+ * @since 3.0
+ *
+ * @param string $content The raw post content to be filtered.
+ * @return string Converted content with `srcset` and `sizes` added to images.
+ */
+function tevkori_filter_content_images( $content ) {
+	// Only match images in our uploads directory.
+	$uploads_dir = wp_upload_dir();
+	$path_to_upload_dir = $uploads_dir['baseurl'];
```

Comment:
Hey @bradt, thanks for the input! I'm curious when we would need a filter here rather than using the filter already present in `wp_uploads_dir()`. Do you have a specific case in mind?